lblod/frontend-gelinkt-notuleren
# Add configuration per administrative unit

### Overview
Add a way of adding configuration per administrative unit, in the future this will be replaced by user preferences and a call to the backend.
Also added the config needed to test https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/515

##### connected issues and PRs:
GN-5235
https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/515


### Setup
None

### How to test/reproduce
Login as Gemeente Gent, notice the style of the article titles is the old one. Then login as any other admin unit and notice we have the new article titles in there

### Challenges/uncertainties
None



### Checks PR readiness
